test/run_test.py

```diff
@@ -82,20 +83,18 @@
 
 WINDOWS_BLACKLIST = [
     'distributed',
-    'rpc_fork',
```

Comment:
any reason these are removed from the windows blacklist?

Reply:
@rohan-varma 
Wanted to include changes in https://github.com/pytorch/pytorch/pull/29827 to avoid that PR rebasing again.
I should have removed line 71, 73 too.